src/Tizen.NUI.Components/Controls/Button.Internal.cs

```diff
@@ -289,6 +289,11 @@ public override void OnRelayout(Vector2 size, RelayoutContainer container)
         {
             Debug.Assert(size != null);
 
+            if (size.Equals(this.size))
```

Comment:
could you check the warning below?
I know there is `Debug.Assert(size != null);` above, maybe checker can not catch it.
could you change it as `size?.Equals(...` or suppress the warning?

Reply:
Done. Thank you.